css/global/tipografia.css

```diff
@@ -0,0 +1,4 @@
+:root {
+  --font-principal: Comfortaa, sans-serif;

```

Comment:
Eu acredito que essa primeira fonte não seja de sistema, onde você esta carregando ela?

Reply:
achei. Talvez você prefira carregar ela nesse modo: https://stackoverflow.com/questions/14676613/how-to-import-google-web-font-in-css-file